catalog/transformer/catalog-transformer-csv-queryresponse/src/main/java/ddf/catalog/transformer/csv/CsvMetacardTransformer.java

```diff
@@ -43,12 +46,21 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
 
     Map<String, String> aliases =
         (Map<String, String>) arguments.getOrDefault("aliases", new HashMap<>());
-
+    String attributeString = (String) arguments.get("columnOrder");
+    ArrayList<String> attributes =
```

Comment:
I think it's good practice to declare these as `List`s

Reply:
okay will do 👍